pkg/kv/kvserver/store_gossip.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 	"github.com/cockroachdb/errors"
 	"github.com/cockroachdb/redact"
+	"go.uber.org/atomic"
```

Comment:
Was this intentional? Seems like we probably want `sync.Atomic`

Reply:
Oh, nice catch! That was Goland. There are a bunch more, fixed all of them.